src/tests/unit/common/test_3Dpoint.cpp

```diff
@@ -0,0 +1,18 @@
+
+
+// Include our own code
+#include <common/3Dpoint.hpp>
+//#include <cmath>
+// Include the gtest library
+#include <gtest/gtest.h>
+
+// Create our own test to check the different function of our class
+// This function will take two arguments:
+// * The test case name 
+// * The test name
+TEST(first_coordinates_values, second_coordinate_values) {
+   Geometry coordinates(20.0, 45.5, 90.0);
```

Comment:
it would be better to affect the input to some variable and check them. it makes the type explicit for other developers

```cpp
float x = 20, y= 45.5, z = 90; // no need for dot now since that they are specifically float

ASSERT_FLOAT_EQ(coordinates.x, x);
ASSERT_FLOAT_EQ(coordinates.y, y);
ASSERT_FLOAT_EQ(coordinates.z, z);
```